src/admin/components/AddAttendanceForm/index.tsx

```diff
@@ -1,14 +1,11 @@
-import React, { useState, ChangeEventHandler, FocusEventHandler, FormEventHandler } from 'react';
-import { AutoComplete, Checkbox, Form, Input, Button, Tooltip, Tag, Icon, Select } from 'antd';
+import { AutoComplete, Button, Checkbox, Form, Icon, Input, Select, Tag, Tooltip } from 'antd';
+import React, { FormEventHandler, useState } from 'react';
```

Comment:
Typically, I think in React the 'react' import should always be first in file (unless you're in Next.js, where you don't need it). I might wrong on this, and CRA might have removed that need. We should readjust that in files where it happens.

Reply:
[same as above](https://github.com/acmucsd/membership-portal-ui/pull/550#discussion_r837275349). I don't believe the react import needs to come first, but it's more idiomatic. Also, I think you're referring to the [new JSX Transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) where the `React` import is not needed and JSX completely handled at compile time. Unless we need to use React 16.3, we could upgrade to 16.4 or 17 to take advantage of it. Facebook even provides a [codemod](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports) to automatically remove the unneeded imports. All we'd have to do is upgrade `react` and `react-scripts` I believe.